app/services/DateService.scala

```diff
@@ -46,10 +46,31 @@ class DateService @Inject()(clock: Clock) {
     startOfNextQuarter.minusDays(1)
   }
 
+  def startOfCurrentQuarter: LocalDate = {
+    startOfNextQuarter.minusMonths(3)
+  }
+
   def lastDayOfMonthAfterCalendarQuarter: LocalDate = {
     startOfNextQuarter.withDayOfMonth(startOfNextQuarter.lengthOfMonth)
   }
 
+  def isDOFSDifferentToCommencementDate(dateOfFirstSale: Option[LocalDate], commencementDate: LocalDate): Boolean = {
+    if(dateOfFirstSale.isDefined) dateOfFirstSale.get != commencementDate else false
+  }
+
+  def getVatReturnEndDate(commencementDate: LocalDate): LocalDate = {
```

Comment:
Do we need these methods if we know already when the date will be? IE: dateService. startOfNextQuarter or dateService. lastDayOfCalendarQuarter